app/main.py

```diff
@@ -1,8 +1,52 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
```

Comment:
Please fix so that every parameter starts on a separate line:
```
def __init__(
    self, 
    comfort_class: int...
or
def __init__( self, 
              comfort_class: int...